streampipes-extensions/streampipes-sinks-brokers-jvm/pom.xml

```diff
@@ -86,7 +86,6 @@
         <dependency>
             <groupId>io.nats</groupId>
             <artifactId>jnats</artifactId>
-            <version>${nats.version}</version>
```

Comment:
I assume the changes to the POM files are comitted by accident, right?

Reply:
when i run osv-scanner for scanning our repo initially in my local machine, these are the errors it was throwing:

```
Failed to resolve version of io.nats:jnats: property "nats.version" could not be foundScanned /home/ibergx00/Documents/oss/streampipes/streampipes-extensions/streampipes-sinks-brokers-jvm/pom.xml file and found 18 packages

Failed to resolve version of com.google.inject:guice: property "guice.version" could not be foundScanned /home/ibergx00/Documents/oss/streampipes/streampipes-maven-plugin/pom.xml file and found 18 packages

Failed to resolve version of org.apache.flink:flink-connector-kafka_2.11: property "flink.version" could not be found
Failed to resolve version of org.apache.flink:flink-java: property "flink.version" could not be found
Failed to resolve version of org.apache.flink:flink-streaming-java_2.11: property "flink.version" could not be found
Failed to resolve version of org.apache.flink:flink-clients_2.11: property "flink.version" could not be found Scanned /home/ibergx00/Documents/oss/streampipes/streampipes-wrapper-flink/pom.xml file and found 9 packages

```

after i made those changes in `pom.xml`, error get resolved

Reply:
after i made changes to pom.xml, i ran `mvn clean package` it was successfully build.

Reply:
Hm I see
can you rebase your branch with the current dev branch and try it again?
After this, the maven build should go through without any issues
At least we have neither a problem in the CI runs nor on my local machine (just verified it)

Reply:
Done

Reply:
Great, in this case we can remove the changes to the POM files from this PR

Reply:
Done :+1: